packages/shared/src/graphql/squads.ts

```diff
@@ -82,6 +86,31 @@ export const CREATE_SQUAD_MUTATION = gql`
       public
       type
       description
+      members {
+        edges {
+          node {
+            referralToken
+          }
+        }
+      }
+    }
+  }
```

Comment:
Why do you retrieve the token on createSquad, do we need it at that point?

Reply:
Yes, this is when the user will be copying the invitation link.